### PR TITLE
Jv 81 style effectiveness label

### LINKED ISSUE #81: Style the effectiveness label
**Is your feature request related to a problem? Please describe.**
Style the effectiveness label so that the user can recognize effective moves versus ineffective moves based on Pokemon type.

**Describe the solution you'd like**
Use CSS to style label.

### PR DESCRIPTION
**Pull request by Wolftownn**
## Changes
_List Changes Introduced by this PR_
1. Styles the effectiveness labels for moves. 

## Purpose
Lets the user tell between effective and ineffective moves easily.

## Approach
By making the colors on each label different, it gives the user an easier way to tell between effective and ineffective moves.

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #81

### CODE REVIEW

**Review by Lap343 — APPROVED** [review]
Nice styling. Looks good

File: pokemon-app/src/styles/MovesList.css [hunk 1]
```diff
@@ -244,3 +244,38 @@
     top: 38.4%;
   }
 }
+
+
+/* Styling for effectiveness label */
+.effective-label {
+  box-sizing: border-box;
```
Comment (gonzalespaulb) [inline pokemon-app/src/styles/MovesList.css:251]: I suggest making this global by using   * {box-sizing:border-box} so it applies to all of your elements

File: pokemon-app/src/components/MovesList.js [hunk 2]
```diff
@@ -53,11 +53,12 @@ const MovesList = ({
...
-            <li key={moveIndex}>
+            <li className="list-spacing-moves" key={moveIndex}>
               {moveData.move.name}
 
               {effectivenessArray && effectivenessArray.length && (
-                <span className="effective-label">
+                <span className="effective-label" data-value={`${effectivenessArray[moveIndex]}`}
+                >
                   {effectivenessArray[moveIndex]}
                 </span>
```
Comment (JayDampitan) [inline pokemon-app/src/components/MovesList.js:63]: Sweet conditional rendering!

File: pokemon-app/src/styles/MovesList.css [hunk 3]
```diff
@@ -244,3 +244,38 @@
...
+  color: #fefefe;
+  width: 4.2rem;
+  background-color: rgb(65, 95, 165);
+}
+
+.list-spacing-moves {
+  margin: 8px !important;
+}
+
+.list-spacing-moves > span:empty {
```
Comment (gonzalespaulb) [inline pokemon-app/src/styles/MovesList.css:279]: Next level css!!

**Review by gonzalespaulb — APPROVED** [review]

**Review by dannysanchez559 — APPROVED** [review]
cant wait to see your guys' styling

**Review by JayDampitan — APPROVED** [review]
Awesome styling! Keep up the great work fam!

**Review by connielion — APPROVED** [review]
Looks good! And agreed, Paul.

File: pokemon-app/src/styles/MovesList.css (hunk 1 above)
Comment (Wolftownn) [reply pokemon-app/src/styles/MovesList.css:251]: Thanks Paul. Will definitely take that into account!

**Pull request merged**